src/System.IO.Pipes.AccessControl/tests/NamedPipeTests/NamedPipeTest.AclExtensions.cs

```diff
@@ -44,6 +44,7 @@ public void GetAccessControl_NamedPipe_BeforeWaitingToConnect()
         /// Tests that SetAccessControl on a Broken NamedPipeClientStream throws an Exception.
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "On Desktop we throw an exception based on the internal PipeStream state. In .NET Core, we do not have access to that member since Pipes.AccessControl is its own assembly.")]
```

Comment:
I assume it is only `Assert.Throws<InvalidOperationException>(() => client.SetAccessControl(new PipeSecurity()));` that fails on .NET Core? If that is the case I don't understand why we would skip this test on NETFramework. 

Also if that is the only line that fails we should make sure the rest of the coverage is covered for desktop as well.